llvm/lib/ObjCopy/MachO/MachOLayoutBuilder.cpp

```diff
@@ -116,11 +116,10 @@ uint64_t MachOLayoutBuilder::layoutSegments() {
   const bool IsObjectFile =
       O.Header.FileType == MachO::HeaderFileType::MH_OBJECT;
   uint64_t Offset = IsObjectFile ? (HeaderSize + O.Header.SizeOfCmds) : 0;
-  if (O.EncryptionInfoCommandIndex) {
-    // If we are emitting an encryptable binary, our load commands must have a
-    // separate (non-encrypted) page to themselves.
-    Offset = alignToPowerOf2(HeaderSize + O.Header.SizeOfCmds, PageSize);
-  }
+  // If we are emitting an encryptable binary, our load commands must have a
+  // separate (non-encrypted) page to themselves.
+  bool FirstSectExtraEncryptableOffset =
```

Comment:
nit: `FirstSectExtraEncryptableOffset` sounds like `uint64_t` rather than `bool`
```suggestion
  bool RequiresPageAlignedSectionOffset =
```

Reply:
I chose a different spelling, but still using `Requires…`, which should be clearer this was a boolean. Thanks for the feedback!